### PR TITLE
Clarify why processing @context is useful for JSON-based processors.

### PR DESCRIPTION
**Pull request by msporny**
Related to issue #491.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/546.html" title="Last updated on Apr 24, 2019, 2:40 AM UTC (0221fbb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/546/8fcd004...0221fbb.html" title="Last updated on Apr 24, 2019, 2:40 AM UTC (0221fbb)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2098,6 +2098,17 @@ <h4>Semantic Interoperability</h4>
...
           </ul>
 
+          <p>
+If implementers follow the requirements above, they are guaranteed semantic
+interoperability between JSON and JSON-LD syntaxes for terms defined by the
+<code>@context</code> mechanism. This is because the exact terms that a
+producer and consumer of the data model described by this specification,
+as well as any extensions used, are explicitly stated through the use and
+processing of the <code>@context</code> property. In other words, the context
+in which the data exchange happens is explictly stated as opposed to implicitly
```
Comment (dlongley) [inline index.html]:
```suggestion
in which the data exchange happens is explicitly stated as opposed to implicitly
```

File: index.html [hunk 2]
```diff
@@ -2098,6 +2098,17 @@ <h4>Semantic Interoperability</h4>
             </li>
           </ul>
 
+          <p>
+If implementers follow the requirements above, they are guaranteed semantic
+interoperability between JSON and JSON-LD syntaxes for terms defined by the
+<code>@context</code> mechanism. This is because the exact terms that a
+producer and consumer of the data model described by this specification,
+as well as any extensions used, are explicitly stated through the use and
+processing of the <code>@context</code> property. In other words, the context
```
Comment (dlongley) [inline index.html]: I had trouble parsing the sentence starting with `This is because...` ... I think a word was forgotten or something?

**Comment by msporny** [comment]
> I raised #548 as an alternative for the same thing because a line-by-line commentary turned out to be a difficult way to express what I thought might be improved. Feel free to merge either one and close the other.

I like some of the language in your PR better than the language in this one. I'll try to merge PR #548 with this one where it makes sense to do so.

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html]: Fixed in ad5c9bcd1b2b956fcb0ccc7749d8a2915e1b8fd5.

**Comment by msporny** [comment]
Applied all changes requested by @dlongley. Merged PR #548 from @chaals into this PR. Multiple positive independent reviews. Editorial. Merging.

**Comment by David-Chadwick** [comment]
I have one question about this text which is, how does a JSON processor find the human readable document that is referred to? The text only says that the machine-readable description should exist at the URL in the @context. Wouldn't it be better to say that the human readable version should also be at the same URL, perhaps as comments in the JSON-LD.

This is the text I am questioning:

A human-readable document describing the expected order of values for the
<code>@context</code> <a>property</a> is expected to be published by any
implementer seeking interoperability. A machine-readable description
(i.e., a normal JSON-LD Context document) is expected to be published
at the URL specified in the <code>@context</code> <a>property</a> by
JSON-LD implementers seeking interoperability.

**Comment by msporny** [comment]
@David-Chadwick can you please open a new issue on the item you placed above.

**Comment by David-Chadwick** [comment]
Done. You can close this PR now

**Pull request merged**